glue/index.js

```diff
@@ -31,55 +34,96 @@ export class Asdom {
 				let el =
 					tag === 'body'
 						? document.body || thro('bug!')
-						: this.wasmImports.asdomImports.documentCreateElement(docId, tag)
+						: this.wasmImports.asDOM_Document.documentCreateElement(docId, tag)
 				this.__refs.set(elId, el)
 			},
 			getElement: id => {
 				return this.__refs.get(id)
 			},
 
-			logusize: log,
-			logstring: s => {
-				log(this.__getString(s))
-			},
-			logstrnull: s => {
-				log(this.__getString(s))
-			},
-
 			documentCreateElement: (id, tag) => {
 				const document = this.__refs.get(id)
 				return document.createElement(tag)
 			},
-
 			documentHasBody: id => {
 				const document = this.__refs.get(id)
 				return document.body ? true : false
 			},
-
+		},
+		asDOM_Element: {
+			// element.setAttribute
 			elSetAttribute: (id, attr, value) => {
```

Comment:
As we will have these more specific namespaces and separate files, perhaps then we can just rename the functions, f.e. just setAttribute instead elSetAttribute. Wdyt?